keva/server/src/test/java/com/jinyframework/keva/server/SnapshotServiceTest.java

```diff
@@ -0,0 +1,134 @@
+package com.jinyframework.keva.server;
+
+import com.jinyframework.keva.server.core.Server;
+import com.jinyframework.keva.server.core.SnapshotConfig;
+import com.jinyframework.keva.server.util.SocketClient;
+import lombok.val;
+import lombok.var;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.util.concurrent.TimeUnit;
+
+import static com.jinyframework.keva.server.util.PortUtil.getAvailablePort;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class SnapshotServiceTest {
+    static String host = "localhost";
+    static String snapInterval = "PT2S";
+
+    Server startServer(int port) throws Exception {
+        val snapshotConfig = SnapshotConfig.builder()
+                .snapshotInterval(snapInterval)
+                .recoveryPath("./dump.keva")
+                .build();
+        val server = Server.builder()
+                .host(host)
+                .port(port)
+                .snapshotConfig(snapshotConfig)
+                .build();
+        new Thread(() -> {
+            try {
+                server.run();
+            } catch (Exception e) {
+                fail(e);
+            }
+        }).start();
+
+        // Wait for server to start
+        TimeUnit.SECONDS.sleep(1);
+        return server;
+
```

Comment:
Remove trailing space

Reply:
wtf how u even notice this shit

Reply:
Please follow our [Developer Guide -> Think aesthetics section](https://jinyframework.com/guide/developer-guide.html#think-aesthetics)

Reply:
![image](https://user-images.githubusercontent.com/23312088/106989138-08828100-67a4-11eb-84d5-fd6634bd131e.png)